scripts/openroad/common/resizer.tcl

```diff
@@ -46,4 +46,16 @@ proc unset_dont_touch_rx {net_pattern} {
             unset_dont_touch "$net_name"
         }
     }
-}
\ No newline at end of file
+}
+
```

Comment:
Not fond of having four functions for this, too much potential for human error. There should be two functions that handle both the RX and the list.